dom-manipulation/main.js

```diff
@@ -0,0 +1,21 @@
+var numberOfClicks = 0;
+var hotButton = document.querySelector('.hot-button');
+var clickCount = document.querySelector('.click-count');
+hotButton.addEventListener('click', hotbtn);
+function hotbtn(event) {
+  numberOfClicks++;
+  clickCount.textContent = numberOfClicks;
+  if (numberOfClicks < 4) {
+    hotButton.className += ' cold';
+  } else if (numberOfClicks > 3 && numberOfClicks < 7) {
+    hotButton.className += ' cool';
```

Comment:
nice work on getting this going. The one disadvantage to this approach is that you will end up with many classes on the element at the end rather than just the hot button and color class.